pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1016,6 +1016,15 @@ func TestNodesWherePreemptionMightHelp(t *testing.T) {
 			},
 			expected: map[string]bool{"machine1": true, "machine3": true},
 		},
+		{
+			name: "ErrReasonNodeLabelNotMatch should not be tried as it indicates that the pod is unschedulable due to node doesn't have the required label",
+			nodesStatuses: framework.NodeToStatusMap{
+				"machine2": framework.NewStatus(framework.UnschedulableAndUnresolvable, podtopologyspread.ErrReasonNodeLabelNotMatch),
```

Comment:
My recent PR polished this whole test a bit. You may need to rebase first, and change "machine" to "node"; otherwise UT may fail.

Reply:
will do, thanks for the reminder.